### PR TITLE
Update to Angular 13

### PR DESCRIPTION
**Pull request by soyzamudio**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
Supports Angular 13

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts [hunk 1]
```diff
@@ -8,10 +8,9 @@ import { TooltipService } from './tooltip.service';
 import { InjectionService } from './injection.service';
 
 @NgModule({
-  declarations: [TooltipContentComponent, TooltipDirective],
-  providers: [InjectionService, TooltipService],
-  exports: [TooltipContentComponent, TooltipDirective],
-  imports: [CommonModule],
-  entryComponents: [TooltipContentComponent]
```
Comment (Hypercubed) [inline projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts:15]: Is there a specific reason this was removed?

**Review by Hypercubed — COMMENTED** [review]
Changes LGTM... please run prettier so we can see the tests.

File: projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts (hunk 1 above)
Comment (soyzamudio) [reply projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts:15]: i think the CLI update command removed it automatically 🤔

Comment (soyzamudio) [reply projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts:15]: Ahh seems like angular 13 deprecated `entryComponents` and it's no longer needed

Comment (Hypercubed) [reply projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts:15]: We might need to keep it for now... see the note "You may still need to keep these if building a library that will be consumed by a View Engine application."

**Comment by michaelfaith** [comment]
Any thoughts on when this will make it into a release?

**Comment by NashIlli** [comment]
+1

**Comment by luoxiao-xy** [comment]
+1

**Comment by martibenn** [comment]
[v20.0.1](https://github.com/swimlane/ngx-charts/releases/tag/20.0.1) seems available

**Comment by michaelfaith** [comment]
> [v20.0.1](https://github.com/swimlane/ngx-charts/releases/tag/20.0.1) seems available

That's good to know.  Sadly, looking at the Changelog on their site (since they don't have one here): https://swimlane.gitbook.io/ngx-charts/changelog  It doesn't list 20.x at all, or what was included...  Definitely don't seem to be great custodians of this codebase. It's unfortunate, considering what a solid product it is.

**Comment by bbarry** [comment]
changelog is in the repo here: https://github.com/swimlane/ngx-charts/blob/master/docs/changelog.md

seems like master is behind the release though: https://github.com/swimlane/ngx-charts/blob/release/20.0.1/docs/changelog.md

**Comment by michaelfaith** [comment]
@bbarry thanks.  That helps.

**Comment by cfrezier** [comment]
When will this release be available on npmjs official registry ?

**Pull request merged**